sermant-plugins/sermant-flowcontrol/flowcontrol-plugin/src/main/java/io/sermant/flowcontrol/DispatcherServletInterceptor.java

```diff
@@ -71,12 +99,12 @@ private Optional<HttpRequestEntity> convertToHttpEntity(HttpServletRequest reque
      * @param request request information
      * @return headers
      */
-    private Map<String, String> getHeaders(HttpServletRequest request) {
-        final Enumeration<String> headerNames = request.getHeaderNames();
+    private Map<String, String> getHeaders(Object request) {
+        final Enumeration<String> headerNames = getHeaderNames.apply(request);
```

Comment:
The value may be null. You are advised to check the value.

Reply:
in spring framework, headerNames are not null